src/modules/tests/test-status/test-status.effects.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { Effect, Actions, ofType } from '@ngrx/effects';
+import { switchMap, catchError } from 'rxjs/operators';
+
+import * as testStatusActions from './test-status.actions';
+import { TestPersistenceProvider } from '../../../providers/test-persistence/test-persistence';
+import { of } from 'rxjs/observable/of';
+
+@Injectable()
+export class TestStatusEffects {
+  constructor(
+    private actions$: Actions,
+    private testPersistenceProvider: TestPersistenceProvider,
+  ) {}
+
+  @Effect({ dispatch: false })
+  testDecidedEffect$ = this.actions$.pipe(
+    ofType(testStatusActions.TEST_STATUS_DECIDED),
```

Comment:
Where will this action be dispatched from?

Reply:
Sorry, thought I replied to this earlier, but I guess I didn't submit it. Right now, this action is triggered when arriving on the Back to Office screen. There needs to be another trigger for when we defer a test write up from the Office screen.